frontend/src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -101,6 +113,28 @@ export const GroupCard = ({
                             />
                         </StyledHeaderActions>
                     </StyledTitleRow>
+                    <ConditionallyRender
+                        condition={!!group.rootRole}
```

Comment:
```suggestion
                        condition={Boolean(group.rootRole)}
```

Reply:
Happy to take this one, is this a style thing or is there a sharp edge to using `!!` that I don't know about?

Reply:
Update: Answered elsewhere. So accepted!